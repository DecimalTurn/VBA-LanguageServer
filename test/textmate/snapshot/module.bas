Attribute VB_Name = "ModuleName"

Option Explicit
