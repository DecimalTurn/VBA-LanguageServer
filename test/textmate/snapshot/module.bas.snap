>Attribute VB_Name = "ModuleName"
#^^^^^^^^^ source.vba meta.attribute.vba keyword.attribute.vba
#         ^ source.vba meta.attribute.vba
#          ^^^^^^^ source.vba meta.attribute.vba entity.other.attribute-name.vba
#                 ^ source.vba meta.attribute.vba
#                  ^ source.vba meta.attribute.vba keyword.operator.comparison.vba
#                   ^ source.vba meta.attribute.vba
#                    ^^^^^^^^^^^^ source.vba meta.attribute.vba string.quoted.double.vba
>
>Option Explicit
#^^^^^^^^^^^^^^^ source.vba keyword.control.vba
>